name.abuchen.portfolio.tests/src/name/abuchen/portfolio/datatransfer/pdf/solarisbank/SolarisbankAGPDFExtractorTest.java

```diff
@@ -85,19 +58,10 @@ public void testGiroKontoauszug02()
 
         assertThat(errors, empty());
 
-        assertThat(results.stream().filter(i -> i instanceof TransactionItem)
-                        .filter(i -> i.getSubject() instanceof AccountTransaction)
-                        .filter(i -> ((AccountTransaction) i.getSubject()).getType() == AccountTransaction.Type.DEPOSIT)
-                        .count(), is(1L));
+        assertThat(results.stream().filter(TransactionItem.class::isInstance).count(), is(1L));
```

Comment:
remove this line and add this
`        assertThat(errors, empty());
        assertThat(countSecurities(results), is(0L));
        assertThat(countBuySell(results), is(0L));
        assertThat(countAccountTransactions(results), is(1L));
        assertThat(results.size(), is(1));
        new AssertImportActions().check(results, CurrencyUnit.EUR);`